packages/tests/server/react/dehydrate.test.tsx

```diff
@@ -24,40 +21,9 @@ test('dehydrate', async () => {
   expect(dehydrated).toHaveLength(2);
 
   const [cache, cache2] = dehydrated;
-  expect(cache.queryHash).toMatchInlineSnapshot(
-    `"[[\\"allPosts\\"],{\\"type\\":\\"query\\"}]"`,
-  );
-  expect(cache.queryKey).toMatchInlineSnapshot(`
-    Array [
-      Array [
-        "allPosts",
-      ],
-      Object {
-        "type": "query",
-      },
-    ]
-  `);
-  expect(cache.state.data).toEqual(db.posts);
-  expect(cache2.state.data).toMatchInlineSnapshot(`
-    Object {
-      "createdAt": 0,
-      "id": "1",
-      "title": "first post",
-    }
-  `);
-});
-
-test('deprecated alias', async () => {
-  const { db, appRouter } = factory;
-  const ssg = createProxySSGHelpers({ router: appRouter, ctx: {} });
+  // typescript doesn't know that it definitely has 2 elements in the array
+  if (!cache || !cache2) throw Error("can't happen");
```

Comment:
I was getting a type error here, presumably due to `noUncheckedIndexedAccess`? This gets rid of the error but it's terrible, is there a better way of asserting these definitely aren't undefined?